subprojects/dependency-management/src/main/java/org/gradle/api/internal/artifacts/ivyservice/ivyresolve/DynamicVersionResolver.java

```diff
@@ -205,12 +189,7 @@ public boolean canMakeFurtherAttempts() {
         }
 
         void resolve() {
-            cacheLockingManager.useCache(new Runnable() {
```

Comment:
Where did the protection for the cache go? You mentioned you had a question about this, but I couldn't find it.